src/app/contexts/PlatformContext/index.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { node, string, func } from 'prop-types';
 
-const PlatformContext = React.createContext('default');
+export const PlatformContext = React.createContext('default');
```

Comment:
Could you try not exporting this and using the already exported `PlatformContextProvider`. This would be more consistent